cmd/subspace/web.go

```diff
@@ -190,10 +190,17 @@ func WebHandler(h func(*Web), section string) httprouter.Handle {
 
 		// Needs a new session.
 		if samlSP != nil {
-			if token := samlSP.GetAuthorizationToken(r); token != nil {
-				r = r.WithContext(samlsp.WithToken(r.Context(), token))
+			session, _ := samlSP.Session.GetSession(r)
```

Comment:
as singin will be required at the end of the middleware flow, we could just log the error for future reference and continue, because session will be `nil` anyways. Or do you think it would be better to explicitly redirect to `/signin`?

https://github.com/subspacecommunity/subspace/blob/e3fc8118a05bf90cb8640f009067ee05ab8e9281/cmd/subspace/web.go#L237-L238

Reply:
added logging to the error, however, now every time we start the container it logs an error. I will change it to debug instead of error